### ISSUE TITLE
Capability: Display map content in a users preferred language

### ISSUE THREAD

**Issue by nchan0154** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#136]
This capability popped into my head when I was working on adding tags to the various capability conclusions. I noticed that we have several tags related to localization, but no capabilities related to this yet.

Ideally, the map language would reflect the language set in the browser headers. From some quick testing, a few tools seem to support this, Google Maps and Bing being the main ones.

**Comment by prushforth** [comment]
Good catch!  I believe this ties into content language negotiation which is tied as you say to headers, so +1 for conveying map (and language) semantics via content negotiation.

**Comment by nchan0154** [comment]
Bringing you in @AmeliaBR  as it may have gotten lost in the Great Wall of Notifications

**Comment by AmeliaBR** [comment]
This is definitely an important user-focused use case (see map content with labels you can read!)

Might need a little more thought to define how this can be broken out into a technical capability of the map viewer.

For the case of serving a different image file: If it is handled through HTTP content negotiation with the server, different data getting served based on the accept-language headers automatically added by the browser to all requests, then that's pretty transparent for the viewer widget code.  It's the server that has the extra behavior.  Still maybe worth mentioning in the context of data formats and server APIs.

But, if the localized content also includes real text, than the viewer does need to be able to detect the language in the returned feature data and handle it correctly when it's inserted into the markup.
